### PR TITLE
MTM-49308 Minor Doc Review Changes

### PR DESCRIPTION
**Pull request by rohitjoshi-softwareag**
Minor doc review changes from previous releases that did not make their way up.

### CODE REVIEW

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@rohitjoshi-softwareag How about 10.16 and develop? Should these changes not be implemented there as well?

File: content/reference/notifications-bundle/overview.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This subscribed object is known as the source object for the <kdb>notification/e
 Subscriptions are set up using the [subscription method](https://{{<domain-c8y>}}/api/{{< c8y-current-version >}}/#tag/Subscriptions) of the Notifications 2.0 API.
 This API requires the calling user to be an authenticated {{< product-c8y-iot >}} user and to have the new ROLE_NOTIFICATION_2_ADMIN role.
 
-When subscribing to notifications, a filter for notifications can be specified which determines the APIs (alarms, alarms with children, events, events with children, measurements, managed objects, operations or any combination of these) to filter by. The alarms with children and events with children enable users to create explicit subscriptions that allow the delivery of child as well as parent managed object events and alarms.
+When subscribing to notifications, a filter for notifications can be specified which determines the APIs (alarms, alarms with children, events, events with children, measurements, managed objects, operations, operations or any combination of these) to filter by. The alarms with children and events with children enable users to create explicit subscriptions that allow the delivery of child as well as parent managed object events and alarms.
```
Comment (sagscmi) [inline content/reference/notifications-bundle/overview.md:37]: I don't think this is needed; "operations" were already mentioned.  Maybe keep the comma though.

**Review by sagscmi — CHANGES_REQUESTED** [review]

File: content/reference/notifications-bundle/overview.md (hunk 1 above)
Comment (rohitjoshi-softwareag) [reply content/reference/notifications-bundle/overview.md:37]: > @rohitjoshi-softwareag How about 10.16 and develop? Should these changes not be implemented there as well?

Hi, I will review those branches to check if they need any changes and raise PR if required.

Comment (rohitjoshi-softwareag) [reply content/reference/notifications-bundle/overview.md]: I fixed the typo. I probably got it in accidentally due to cherry-picking.

**Review by sagscmi — APPROVED** [review]

**Pull request merged**